bin/DMHS_ATS_configurable

```diff
@@ -45,13 +39,17 @@ def cmdline():
                         help="Port number for webserver")
     parser.add_argument("--send_efd_message", action="store_true", default=False,
                         help="Send and EDF message as well?")
+    parser.add_argument("--log", action="store", default='INFO',
+                         choices=["DEBUG","INFO","WARNING","ERROR","CRITICAL"],
+                         help="Logging Level [DEBUG/INFO/WARNING/ERROR/CRITICAL]")
+
     # args = parser.parse_args()
     # Set the defaults of argparse using the values in the yaml config file
     parser.set_defaults(**conf_defaults)
     args = parser.parse_args(args=remaining_argv)
+    args.loglevel = getattr(logging, args.log.upper())
```

Comment:
The upper casing here is not doing anything because the choices above mandates that only upper case options are allowed. I think you need to add `type=str.upper` to your `add_argument()` call above so that lower case `debug` is converted to `DEBUG` before the choices are checked.